src/modules/LanItems.rb

```diff
@@ -1886,24 +1878,24 @@ def setup_basic_device_options(devmap)
     # @return true if success
     def Commit
       if @operation != :add && @operation != :edit
-        Builtins.y2error("Unknown operation: %1", @operation)
+        log.error("Unknown operation: %1", @operation)
```

Comment:
BUM! :bomb:  crash. `log.error` accept only one argument. Testsuite should catch it ;)